client/src/components/JurisdictionAdmin/generateSheets.ts

```diff
@@ -423,38 +427,52 @@ export const downloadBatchTallySheets = async (
     // Assume up until this point that we won't spill onto a second page. From here onward, no
     // longer make that assumption
 
-    // autoTable automatically adds page breaks
-    autoTable(doc, {
-      head: [['Candidates/Choices', 'Enter Stack Totals']],
-      body: contestChoices.map(choice => [
-        choice.name,
-        '', // Stack totals left blank for the audit board to fill out
-      ]),
-      startY: y,
-      margin: { bottom: pageMargin, left: pageMargin, right: pageMargin },
-      rowPageBreak: 'avoid',
-      theme: 'grid',
-      styles: {
-        cellPadding: tableCellPadding,
-        fillColor: 'white',
-        fontSize: defaultFontSize,
-        fontStyle: 'normal',
-        lineColor: 'black',
-        lineWidth: drawingLineWidth,
-        minCellWidth: tableCellMinWidth,
-        textColor: 'black',
-      },
-      headStyles: {
-        fontStyle: 'bold',
-      },
-    })
+    for (const [contestIndex, contest] of contests.entries()) {
+      // autoTable automatically adds page breaks
+      autoTable(doc, {
+        head: [
+          [{ content: contest.name, colSpan: 2 }],
```

Comment:
A light gray background on the contest name header row might help make it more readable

Reply:
https://github.com/votingworks/arlo/pull/1879/commits/ecfcebbcc0a4d52474d6e6ebeac22ae9d2e06ec8